docs/get-started/upgrading.md

```diff
@@ -1,6 +1,6 @@
 # Upgrading
 
-The Planet SDK for Python is Version 2 of what was previously referred to as
+The Planet Earth Observation Toolkit is Version 2 of what was previously referred to as
```

Comment:
I see we're at least consistent on this, so that's good, but where was the decision about this made?